Authentication/Login/View/LoginConfiguration.swift

```diff
@@ -13,7 +13,7 @@ import Foundation
 */
 public protocol LoginControllerTransitionDelegate {
```

Comment:
If somebody else wants to know when a user has logged in they should use the service for that. This is intended only to customize what this particular screen should do after a user has logged in. We use it to continue with the flow.